fl4health/clients/basic_client.py

```diff
@@ -607,7 +612,7 @@ def set_optimizer(self, config: Config) -> None:
         """
         optimizer = self.get_optimizer(config)
         assert not isinstance(optimizer, dict)
-        self.optimizer = optimizer
+        self.optimizers = {"global": optimizer}
```

Comment:
If we just have a single optimizer, I call it the global one. Maybe this isn't the best name, feel free to suggest something else.

Reply:
I don't have a good alternative. Global is good for now.